tests/languages/bash/var_assign_feature.test

```diff
@@ -0,0 +1,23 @@
+foo=12
+bar+='xyz'
+foo+=('xyz')
+
+----------------------------------------------------
+
+[
+	["assign-left", ["foo"]],
+	["operator", ["="]],
+	["number", "12"],
+	["assign-left", ["bar"]],
+	["operator", ["+="]],
+	["string", ["'xyz'"]],
+	["assign-left", ["foo"]],
+	["operator", ["+="]],
+	["punctuation", "("],
+	["string", ["'xyz'"]],
+	["punctuation", ")"]
+]
+
+----------------------------------------------------
+
+Checks for variable assignments.

```

Comment:
We usually try to have a correspondance between test files' names and patterns in the language definition. So you should rename this file `assign-left_feature.test` or rename the pattern, so that it's more clear what this test file does without even reading it.